nifi-registry-assembly/pom.xml

```diff
@@ -423,5 +423,43 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>include-aws</id>
```

Comment:
I like making this provider an optional extension, but is there any reason not to include it by default (i.e., licensing or binary size).

Or do you think for our Apache convenience binary builds we will include this profile? I think it would be pretty popular. Just wondering if we need to discuss before the next release, and maybe update the RM guide to specify if this profile should be active for convenience binary builds.